experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -528,7 +528,7 @@ describe("editable-tree: editing", () => {
 			});
 
 			// delete method
```

Comment:
thes comments (here and below) also should be updated

Reply:
updated comments to "remove" instead of "delete"